src/Http/Error.php

```diff
@@ -44,14 +44,42 @@ public function __construct(
         int $status,
         string $message = ''
     ) {
-
         $uri     = $request->getURI();
         $baseurl = $uri->getScheme() .'://'. $uri->getAuthority();
 
-        $tpl_data = array(
-                     'message' => $message,
-                     'baseurl' => $baseurl,
-                    );
+        $tpl_data = [];
+
+        $lorisInstance = $request->getAttribute('loris');
+        $user          = $request->getAttribute('user') ?? new \LORIS\AnonymousUser();
+        
+
+        // Add a link to the issue tracker as long as a LORIS Instance object
+        // is present in the request.
+        if (! $user instanceof \LORIS\AnonymousUser
```

Comment:
Why is this check re-added?

Reply:
I removed it but it caused a bunch of errors, e.g. relating to calling `hasPermission()` on `null` farther down the stack. Removing it seems to break everything

Reply:
Without this check, this error occurs for unathenticated users:

Uncaught TypeError: Argument 1 passed to LORIS\\Middleware\\PageDecorationMiddleware::__construct() must be an instance of User, null given, called in /var/www/loris/src/Middleware/ExceptionHandlingMiddleware.php on line 47 and defined in /var/www/loris/src/Middleware/PageDecorationMiddleware.php:18